remc2/sub_main.cpp

```diff
@@ -117827,7 +117817,7 @@ void* sub_83D70_malloc1(int a1)//264d70
 	if ( !v3 || !sub_83E00((int)v3, v4) )
 	  return 0;
 	sub_85350();
-	memset((void*)*v3, 0, v4);//problém
+	memset((void*)*v3, 0, v4);//problÃ©m
```

Comment:
Is your editor configured to use UTF-8? It looks like there are quite a few changes here that seem to be conversions into an ISO-8859 codepage or so. I think it is just comments but maybe it is worth checking the settings.